autoload/elixir/indent.vim

```diff
@@ -65,6 +66,19 @@ function! s:prev_starts_with(context, expr)
   return s:_starts_with(a:context.prev_nb_text, a:expr, a:context.prev_nb_lnum)
 endfunction
 
+function! s:in_embedded_view()
+  let position = getpos('.')
+  let lnum = line('.')
+  let start_lnum = search('\s\+\~\%\(e\|E\|L\|H\)"""', 'bW')
+  if start_lnum == 0
+    return 0
+  endif
+  let end_lnum = search('\s\+"""', 'W')
+  call setpos('.', position)
```

Comment:
This moves the cursor - is this necessary? We do reset the cursor between handler calls so it's not a huge deal, but I'd be curious what this enables

Reply:
(Note this might be superseded by the syntax comment)